templates/tfengine/components/cicd/main.tf

```diff
@@ -43,6 +43,8 @@ locals {
   cloudbuild_sa = "serviceAccount:${data.google_project.devops.number}@cloudbuild.gserviceaccount.com"
   has_scheduled_jobs = anytrue([for env in var.envs : env.triggers.validate.run_on_schedule || env.triggers.plan.run_on_schedule || env.triggers.apply.run_on_schedule])
   has_apply_jobs = anytrue([for env in var.envs : !env.triggers.apply.skip])
+  is_github = var.github.name != ""
+  is_cloud_source_repository = !var.is_github && var.cloud_source_repository.name != ""
```

Comment:
I'm doing this to ensure that github and csr are mutually exclusive since now both are specified on `variables.tf`

There's no way to enforce that rather than doing something like this, and I put preference over github because csr is the one that adds more extra resources.

Reply:
Add a comment saying these two are mutually exclusive